docs/command_line_reference.rst

```diff
@@ -417,10 +417,27 @@ Default value: ``timeout:60``
 Here are a few common examples:
 
 * Enable HTTP compression: ``--client-options="compressed:true"``
-* Enable SSL (e.g. if you have X-Pack Security installed): ``--client-options="use_ssl:true,verify_certs:true"``. Note that you don't need to set ``ca_cert`` (which defines the path to the root certificates). Rally does this automatically for you.
-* Enable SSL with a client key and certificate: ``--client-options="use_ssl:true,verify_certs:true,ca_certs:'/path/to/cacert.pem',client_cert:'/path/to/client_cert.pem',client_key='/path/to/client_key.pem"`` (see also the `Elasticsearch Python client docs <http://elasticsearch-py.readthedocs.io/en/master/index.html#ssl-and-authentication>`_)
 * Enable basic authentication: ``--client-options="basic_auth_user:'user',basic_auth_password:'password'"``. Avoid the characters ``'``, ``,`` and ``:`` in user name and password as Rally's parsing of these options is currently really simple and there is no possibility to escape characters.
 
+**TLS/SSL**
+
+This is applicable e.g. if you have X-Pack Security installed.
+Enable it with ``use_ssl:true``.
+
+**TLS/SSL Certificate Verification**
+
+Server certificate verification is controlled with the ``verify_certs`` boolean. To disable use ``verify_certs:false``.
```

Comment:
Maybe mention that it is turned on by default?

Reply:
Great point, indeed.